spec/features/user_can_view_items_of_a_category_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+feature "category items" do
+  fixtures :categories
+  fixtures :items
+
+  let(:category) {
+    Category.find_by_name("Category B")
+    }
+
+  before(:each) do
+    visit "/"
+    click_link category.name
+  end
+
+  it "can visit a category page" do
+    expect(current_path).to eq(category.dao.prepend('/'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.